godtools/App/Features/AppLanguage/Data-DomainInterface/DownloadToolLanguageRepository.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  DownloadToolLanguageRepository.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 12/18/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class DownloadToolLanguageRepository: DownloadToolLanguageRepositoryInterface {
+    
+    private let downloadedLanguagesRepository: DownloadedLanguagesRepository
+    
+    init(downloadedLanguagesRepository: DownloadedLanguagesRepository) {
+        
+        self.downloadedLanguagesRepository = downloadedLanguagesRepository
+    }
+    
+    func downloadToolLanguage(languageId: String) -> AnyPublisher<Bool, Never> {
+        
+        // TODO: - actually perform download
+        return Future() { promise in
```

Comment:
Hey @rachaelblue once this PR (https://github.com/CruGlobal/godtools-swift/pull/1855) is merged you'll be able to implement translation downloading with progress.
There's also an example snippet in the description with what that might look like.

Reply:
Thank Levi.  Made a follow-up ticket for implementing downloading: https://jira.cru.org/browse/GT-2234